### PR TITLE
Phonetic: Convert to structured

### PR DESCRIPTION
**Pull request by kirkins**
## Description of new Instant Answer, or changes

- Convert phonetic goodie to structured.
- Add a negative test

## Related Issues and Discussions

#1163

## People to notify

@mintsoft 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/phonetic

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Phonetic.pm [hunk 1]
```diff
@@ -57,7 +57,15 @@ handle remainder => sub {
     $_ = lc;
     my @words = split(/\s+/, $_);
     my @phonetics = map { components($_) } @words;
-    return "Phonetic: " . join(" ", @phonetics);
+    my $string_answer = "Phonetic: " . join(" ", @phonetics);
```
Comment (kirkins) [inline lib/DDG/Goodie/Phonetic.pm]: I think I'll change this to this format:

    Phonetic: abc
    Alfa Bravo Charlie

**Comment by mintsoft** [comment]
@kirkins ping me when you're done with alterations and I'll have another look

**Comment by kirkins** [comment]
@mintsoft I'm done for the alternations on this one. I just added the word at the top as title and the phonetic below. Open to suggestions for format.

**Comment by mintsoft** [comment]
@kirkins LGTM overall, however I think the subtitle and title are probably the wrong way around : 
screenshot_2017-07-17_23-28-09

The answer the user is after should be larger than the question usually

**Comment by mintsoft** [comment]
@kirkins Ping! Just checking in, are you OK with the changes?

**Comment by kirkins** [comment]
@mintsoft the only change was switching the titles right?

**Comment by mintsoft** [comment]
@kirkins Yes that's it, although it looks like you've missed the tests: https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/268782675#L1193

**Comment by kirkins** [comment]
@mintsoft oops that should fix it.

**Comment by mintsoft** [comment]
Excellent, LGTM @kirkins

**Pull request merged**